### PR TITLE
feat: Split Main map into parts

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

**Comment by kristofbolyai** [comment]
https://github.com/Wynntils/Data-Storage/pull/34

File: maps/map-main-5-3.png
Comment (magicus) [inline maps/map-main-5-3.png:1]: This splits Detlas over two parts. :-( It means more work when updating for seasonal Detlas. Can you try to offset the split one region to the east or west?

Comment (kristofbolyai) [reply maps/map-main-5-3.png:1]: Well, then we end up with non-regular sized chunks. Do you want that?

Comment (kristofbolyai) [reply maps/map-main-5-3.png:1]: Also, I would not call it more work, the script in the other PR fully supports creating these.

**Review by magicus — APPROVED** [review]

**Pull request merged**